docs/install-upgrade.md

```diff
@@ -36,12 +35,81 @@ You can deploy the MongoDB Community Kubernetes Operator with different scopes b
 
 You scope the Operator to a namespace. The Operator watches MongoDBCommunity resources in that same [namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/).
 
-This is the default scope when you install the Operator using the [installation instructions](#procedure).
+This is the default scope when you [install the Operator using Helm](#install-the-operator-using-helm) or [install the Operator using kubectl](#install-the-operator-using-kubectl).
 
 #### Operator in Different Namespace Than Resources
 
 You scope the Operator to a namespace. The Operator watches MongoDBCommunity resources in other namespaces.
 
+To deploy the Operator in a different namespace than the resources, [Install in a Different Namespace using Helm](#install-in-a-different-namespace-using-helm) or [Install in a Different Namespace using kubectl](#install-in-a-different-namespace-using-kubectl).
```

Comment:
I'm not sure that the linked section describe how to configure the operator to watch other namespaces with a helm install. I think the helm command would look something like:

```
helm install community-operator mongodb/community-operator --set operator.watchNamespace="dev-dbs"
```

see https://github.com/mongodb/helm-charts/blob/main/charts/community-operator/values.yaml#L16 and https://www.mongodb.com/docs/kubernetes-operator/stable/tutorial/set-scope-k8s-operator/

the kubectl section look OK to me

Reply:
Is watching the namespace a required step? I would think installing in this namespace also watches it; if not, how is installing it in this namespace different from installing in another? The [instructions here](https://github.com/mongodb/helm-charts/tree/main/charts/community-operator) that we're moving over only give the command I included to install in another namespace, but if they must go together I will add this. But I would want to include some information on why/how the two things are different, which I'm not really sure of. Welcoming input from tech reviewer.

Reply:
by default the operator watches only the namespace that it's installed to. If you want to install the operator and your databases in different namespaces, you must instruct the operator which namespaces it should watch. So this step is required only if this is the case.

Reply:
Sorry for the delay! Thanks for the explanation, I see why they are different now and why this might be confusing. I added the command to that section.